infrastructure/tests/test_data_import.py

```diff
@@ -6,14 +6,76 @@
 from django.contrib.admin.sites import AdminSite
 from django_q.models import OrmQ
 import rest_framework.response
+import xlrd
 
-from infrastructure.models import FinancialYear, QuarterlySpendFile, AnnualSpendFile, Expenditure, Project
+from infrastructure.models import FinancialYear, QuarterlySpendFile, AnnualSpendFile, Expenditure, Project, BudgetPhase
 from infrastructure.tests import utils
 from infrastructure.utils import load_excel
 from infrastructure.upload import process_document
 from scorecard.models import Geography
 
 
+def generate_mock_data():
```

Comment:
```suggestion
def generate_mock_project():
```

the word "data" almost never contributes anything in a variable name.